spec/system/zendesk_ticket_deletion_spec.rb

```diff
@@ -137,4 +147,23 @@ def and_i_should_be_able_to_export_tickets_to_csv
     click_on("Export to CSV")
     expect(download_content).to eq(ZendeskDeleteRequest.to_csv)
   end
+
+  def when_i_import_tickets_from_csv
+    click_on("Import from CSV")
+    attach_file(
+      "ZendeskDeleteRequest CSV",
+      Rails.root.join("spec/support/zendesk_delete_requests.csv")
+    )
+    click_on("Upload")
+  end
+  alias_method :when_i_import_the_same_tickets_from_csv,
+               :when_i_import_tickets_from_csv
+
+  def then_i_should_see_the_imported_delete_request
+    expect(page).to have_content("Ticket 42")
```

Comment:
@felixclack I am wondering if for this test we should use another ticket number than 42. This ticket number is already used in the previous tests and should already be present on the page which may suggest that the upload is not working within the test